examples/README.md

```diff
@@ -70,7 +70,13 @@ Graph modeling
 The following code bases use Flax and provide training frameworks and a wealth
 of examples, in many cases with pre-trained weights:
 
-- https://github.com/google-research/scenic: *Scenic* is a codebase/library
+- [HuggingFace Transformers](https://github.com/huggingface/transformers) is a library for bulding, training, and deploying
+  state of the art models powered by the reference open source in machine learning.
```

Comment:
"powered by the reference open source in machine learning" ... this sentence sounds a bit weird; also I do agree that HF is a very important and widely used library with tons of greatness, but I wouldn't go as far as calling it the "reference OSS in ML"

Reply:
Ok I agree, I actually copied this from their website, I now toned it down a bit 😃 .